packages/react/ds/src/header/components/header-menu.tsx

```diff
@@ -16,19 +19,107 @@ type HeaderMenuProps = {
     label?: string;
     icon?: IconId;
   };
+} & Pick<HeaderProps, 'tools'>;
+
+type MenuItemAccordionProps = {
+  index: number;
+  item: { label?: string; slot: React.ReactNode };
+};
+
+type MenuListItemProps = {
+  href?: string;
+  label?: string;
+  bold?: boolean;
+};
+
+const MenuListItem = ({
+  href = '#',
+  label = '',
+  bold = true,
+}: MenuListItemProps) => (
+  <a
+    aria-label={label || 'link with no label'}
+    href={href}
+    className="hover:gi-underline hover:gi-underline-offset-sm gi-block gi-py-4 gi-border-gray-100 gi-border-b-xs gi-border-solid"
+  >
+    <span className={cn('gi-text-sm', 'gi-ml-1', { 'gi-font-bold': bold })}>
+      {label}
+    </span>
+  </a>
+);
+
+export const MenuItemAccordion = ({ index, item }: MenuItemAccordionProps) => {
```

Comment:
Why an `accordion`?

Reply:
Afeter some discussions with Lexa we came out with this solution, just for mobile in this case: https://www.figma.com/design/ZEqfpXhocdZs1K5YfsE5ws/Design-system?node-id=3412-1644&node-type=canvas&m=dev

this way if the user wants, the user can include the prop `keepOnMobile` and if it is true we show the slots in mobile/tablet